### PR TITLE
Added rules for Kayak

### PR DESCRIPTION
**Pull request by kaistian**

### CODE REVIEW

**Comment by TheNavigat** [comment]
LGTM

File: src/chrome/content/rules/Kayak.xml [hunk 1]
```diff
@@ -1,10 +1,77 @@
...
+	<target host="kayak.com.tr" />
+	<target host="www.kayak.com.tr" />
 
+	<target host="ca.kayak.com" />
+	<target host="www.ca.kayak.com" />
+	<target host="cn.kayak.com" />
+	<target host="www.cn.kayak.com" />
+	<target host="fi.kayak.com" />
+	<target host="www.fi.kayak.com" />
+	<target host="gr.kayak.com" />
```
Comment (loveithateit) [inline src/chrome/content/rules/Kayak.xml]: Cert doesn't match ^gr.kayak.com, please rewrite to www.gr.kayak.com

Comment (kaistian) [inline src/chrome/content/rules/Kayak.xml]: @loveithateit I'm not sure what you mean by "rewrite to", but would adding
`<rule from="^http://gr.kayak.com" to="https://www.gr.kayak.com" />`
solve it?

Comment (loveithateit) [inline src/chrome/content/rules/Kayak.xml]: @kaistian yes. Perhaps 'map' would have been clearer.

**Comment by semenko** [comment]
Closing since this has some outstanding work and hasn't been touched in a while. Feel free to re-open if you'd like this merged. :+1:

**Pull request closed without merging**